src/ui/public/vis/agg_config_result.js

```diff
@@ -3,7 +3,7 @@ define(function () {
 
   function AggConfigResult(aggConfig, parent, value, key) {
     this.key = key;
-    this.value = value;
+    this.value = value === null ? false : value; // restrict to primitive for sorting in Angular
```

Comment:
I like this suggestion, and it fixes the bug just as well.

Any thoughts on what value should  be returned? `false` was an arbitrary decision. I'm also leaning towards a hyphen string `'-'`, because I believe that's what is shown in Discover.